packages/common/src/operations.graphql

```diff
@@ -394,77 +312,28 @@ query masterLists(
               unitName
               name
               isVisible
-              availableBatches {
-                ... on ConnectorError {
-                  __typename
-                  error {
-                    ... on PaginationError {
-                      __typename
-                      description
-                      rangeError {
-                        description
-                        min
-                        max
-                        field
-                      }
-                    }
-                    ... on DatabaseError {
-                      __typename
-                      description
-                      fullError
-                    }
-                    description
-                  }
-                }
-                ... on StockLineConnector {
+              availableBatches(storeId: $storeId) {
```

Comment:
`ConnectorError` gone for nested fields also, but the nested fields just return the `XConnector`, not the `XResponse`.. which means we don't use the `... on XConnector` syntax and we also don't need the guard